src/pds/registrysweepers/utils/productidentifiers/pdsproductidentifier.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pds.registrysweepers.utils.productidentifiers.pdslid import PdsLid
+
+from abc import ABC
+
+
+class PdsProductIdentifier(ABC):
+    LIDVID_SEPARATOR = "::"
+
+    @property
+    def lid(self) -> PdsLid:
+        raise NotImplementedError
```

Comment:
Since this already drives from `ABC`, why not use `@abstractmethod` instead of raising `NotImplementedError`?

Reply:
See earlier comment about mediocrity.

will fix